delta/sdk/src/test/scala/ch/epfl/bluebrain/nexus/delta/sdk/resources/ResourcesSpec.scala

```diff
@@ -41,77 +46,104 @@ class ResourcesSpec
     with Fixtures {
 
   "The Resources state machine" when {
-    implicit val sc: Scheduler = Scheduler.global
-    val epoch                  = Instant.EPOCH
-    val time2                  = Instant.ofEpochMilli(10L)
-    val subject                = User("myuser", Label.unsafe("myrealm"))
-    val caller                 = Caller(subject, Set.empty)
+    val epoch   = Instant.EPOCH
+    val time2   = Instant.ofEpochMilli(10L)
+    val subject = User("myuser", Label.unsafe("myrealm"))
+    val caller  = Caller(subject, Set.empty)
 
-    val project    = ProjectGen.resourceFor(ProjectGen.project("myorg", "myproject", base = nxv.base))
-    val projectRef = project.value.ref
+    import ch.epfl.bluebrain.nexus.delta.sdk.resources.ResourceFixture._
 
-    val schemaSource = jsonContentOf("resources/schema.json").addContext(contexts.shacl, contexts.schemasMetadata)
-    val schema1      = SchemaGen.schema(nxv + "myschema", projectRef, schemaSource)
-    val schema2      = SchemaGen.schema(nxv + "myschema2", projectRef, schemaSource)
+    val jsonld = JsonLdResult(myId, compacted, expanded, remoteContexts)
 
-    val fetchSchema: (ResourceRef, ProjectRef) => FetchResource[Schema] = {
-      case (ref, _) if ref.iri == schema2.id => UIO.some(SchemaGen.resourceFor(schema2, deprecated = true))
-      case (ref, _) if ref.iri == schema1.id => UIO.some(SchemaGen.resourceFor(schema1))
-      case _                                 => UIO.none
-    }
-
-    val resourceValidator = new ValidateResourceImpl(ResourceResolutionGen.singleInProject(projectRef, fetchSchema))
```

Comment:
I changed the way that the tests were working here so as not to depend on all the Json-LD process anymore.
It makes things slightly simpler and the part with Json-Ld was already tested in `ResourcesImplSpec`.